concepts/40 Angular Components/10 Getting Started/03 Add DevExtreme to an Angular CLI Application/08 Register 3rd-Party Dependencies in Angular CLI 6+/01 JSZip Registration.md

```diff
@@ -1,4 +1,10 @@
-If you want to use the [DataGrid](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Overview/Angular/Light) widget, register the <a href="https://stuk.github.io/jszip/" target="_blank">JSZip library</a> in the `tsconfig.json` file. The widget uses this library for [client-side export to Excel](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/ExportToExcel/Angular/Light).
+The <a href="https://stuk.github.io/jszip/" target="_blank">JSZip library</a> is useful if you use the following functionality in the [DataGrid](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Overview/Angular/Light) or [FileManager](https://js.devexpress.com/Demos/WidgetsGallery/Demo/FileManager/Overview/Angular/Light) widgets:
```

Comment:
I double-checked and it turns out that JSZip is always required, regardless of the widgets or features you use.

```suggestion
Register the <a href="https://stuk.github.io/jszip/" target="_blank">JSZip library</a> in the `tsconfig.json` file:
```